webapp/src/api/api.ts

```diff
@@ -0,0 +1,49 @@
+import {PlaceType} from "../types/PlaceType";
+
+export async function getPlaces() : Promise<PlaceType[]> {
+    const apiEndPoint = "http://localhost:5000/api"
```

Comment:
We have to take into account that may be in the future this url must be changed as the Web app would be deployed in a machine

Reply:
Yes, we might take into consideration Github Secrets or encription among other alternatives.